sentences/de/person_HassGetState.yaml

```diff
@@ -0,0 +1,121 @@
+language: de
+intents:
+  HassGetState:
+    data:
+      # https://www.home-assistant.io/integrations/person/
+      - sentences:
+          - "<wo_ist> <name>"
+        response: wo
+        requires_context:
+          domain: person
+        slots:
+          domain: person
+
+      - sentences:
+          - "[<befinden>] <name> <zuhause>"
+        response: einzeln_zone
+        requires_context:
+          domain: person
+        slots:
+          domain: person
+          state: home
+
+      - sentences:
+          - "[<befinden>] <name> <nicht_zuhause>"
+        response: einzeln_zone
+        requires_context:
+          domain: person
+        slots:
+          domain: person
+          state: not_home
+
+      - sentences:
+          - "[<befinden>] <name> [<lokale_praeposition>] {zone:state}"
+        response: einzeln_zone
+        requires_context:
+          domain: person
+        excludes_context:
+          state: home
+        slots:
+          domain: person
+
+      - sentences:
+          - "<befinden> <jemand> <zuhause>"
+        response: irgendeins
+        slots:
+          domain: person
+          state: home
+
+      - sentences:
+          - "<befinden> <jemand> <nicht_zuhause>"
+        response: irgendeins
+        slots:
+          domain: person
+          state: not_home
+
+      - sentences:
+          - "<befinden> <jemand> [<lokale_praeposition>] {zone:state}"
+        response: irgendeins
+        slots:
+          domain: person
+
+      - sentences:
+          - "<befinden> <alle> <zuhause>"
+        response: alle
+        slots:
+          domain: person
+          state: home
+
+      - sentences:
+          - "<befinden> <alle> <nicht_zuhause>"
+          - "<befinden> niemand <zuhause>"
+        response: alle
+        slots:
+          domain: person
+          state: not_home
+
+      - sentences:
+          - "<befinden> <alle> [<lokale_praeposition>] {zone:state}"
+        response: alle
+        slots:
+          domain: person
+
+      - sentences:
+          - "wer <befinden> <zuhause>"
+        response: welches
+        slots:
+          domain: person
+          state: home
+
+      - sentences:
+          - "wer <befinden> <nicht_zuhause>"
+        response: welches
+        slots:
+          domain: person
+          state: not_home
+
+      - sentences:
+          - "wer <befinden> [<lokale_praeposition>] {zone:state}"
+        response: welches
+        slots:
+          domain: person
+
+      - sentences:
+          - "<wieviel> (Personen|Menschen) <befinden> <zuhause>"
```

Comment:
i think we can make `(Personen|Menschen)` an expansion rule and add `Leute` to possible values 🙂